settings.gradle

```diff
@@ -11,6 +11,7 @@ dependencyResolutionManagement {
     repositories {
         google()
         mavenCentral()
+        maven { url 'https://jitpack.io' }
```

Comment:
why?

Reply:
we use external library MZip, without this library cannot be downloaded and registered

Reply:
we use external library MZip, without this library cannot be downloaded and registered